internal/grpcapi/repository_server.pb.go

```diff
@@ -1158,6 +1158,8 @@ type FindManifestsRequest struct {
 	unknownFields protoimpl.UnknownFields
 
 	Labels map[string]string `protobuf:"bytes,1,rep,name=labels,proto3" json:"labels,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
+	// if set to non-zero, the server will return up to N entries.
+	PageSize int32 `protobuf:"varint,2,opt,name=page_size,json=pageSize,proto3" json:"page_size,omitempty"`
```

Comment:
Better to use sint32 for efficiency. If we do not expect these to be negative values ever then we can use `uint32`.